README.md

```diff
@@ -1 +1,276 @@
-# bookstore-app-react
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Book store] <a name="Book store"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Book Store]** 
+  This app is a must have instrument for book lovers. It is a webpage that allows you to add your favourite books and authors and save them.
+  It also allows you to remove any books you may have already read or wish to replace.
+  It will accept all books and authors without fail. Built with HTML, CSS, and JS.
+
+## 🛠 Built With <a name="built-with">HTML_CSS_JS_React_Redux</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] Kindly remove every sentence that starts with a `>` as they are meant to be a guide on what to do there. They should be removed after you add the necessary content. 😎️

**Examples**

![Screenshot from 2023-03-01 11-52-28](https://user-images.githubusercontent.com/17243325/222090998-046749d0-7b34-4dbb-bf0e-965912584445.png)
![Screenshot from 2023-03-01 11-52-36](https://user-images.githubusercontent.com/17243325/222091007-5fe7d8b8-3d75-485b-b991-9ec0c93d2a3e.png)